packages/sui-mono/bin/sui-mono-check.js

```diff
@@ -1,6 +1,26 @@
 /* eslint no-console:0 */
+const program = require('commander')
 const checker = require('../src/check')
 
+program
+  .on('--help', () => {
+    console.log('  Description:')
+    console.log('')
+    console.log('    Reports if any package should be released based on ComVer')
+    console.log('')
+    console.log('    It will create a MAJOR only if there are BREAKING CHANGES in the commit description')
```

Comment:
If you don't use the template yes, if you use the template you just need to fill the breaking changes sections.

At the end breaking changes and resolved issues are part of the commit description